python/src/lib.rs

```diff
@@ -15,18 +15,66 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use ballista::prelude::*;
+use datafusion::prelude::*;
+use datafusion_python::context::PySessionContext as DataFusionPythonSessionContext;
+use datafusion_python::utils::wait_for_future;
+use std::collections::HashMap;
+
 use pyo3::prelude::*;
-pub mod context;
 mod utils;
 
-pub use crate::context::PySessionContext;
-
 #[pymodule]
-fn pyballista_internal(_py: Python, m: Bound<'_, PyModule>) -> PyResult<()> {
+fn ballista_internal(_py: Python, m: Bound<'_, PyModule>) -> PyResult<()> {
     pyo3_log::init();
     // Ballista structs
-    m.add_class::<PySessionContext>()?;
+    m.add_class::<PyBallista>()?;
     // DataFusion structs
     m.add_class::<datafusion_python::dataframe::PyDataFrame>()?;
     Ok(())
 }
+
+#[pyclass(name = "Ballista", module = "ballista", subclass)]
+pub struct PyBallista(pub Option<Vec<(String, String)>>);
```

Comment:
config values still not used to configure new context?

Reply:
new updates include the BallistaConfig and some of the getter methods